src/librustdoc/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+  rustdoc::main();
```

Comment:
I think rustfmt won't be happy here.

Reply:
There's another unresolved question - how do we make rustfmt work? I think telling people to use `x.py fmt` should be fine for now.